.github/workflows/trigger_proverbench_dispatch.yml

```diff
@@ -0,0 +1,30 @@
+name: Prover Bench on halo2 PR
+on:
+  pull_request:
+    types: [labeled ,  ready_for_review]
```

Comment:
If I understand this correctly, in order for this workflow to be triggered the PR needs to be ready for review (non-draft), and also have some label, right?  If that's the case, wouldn't it be better to create a specific label to trigger benchmarks?

Reply:
correct. we need to have the PR labeled as "benchmarks". otherwise the job is skipped:

jobs:
  Prover-benches-via-repo-dispatch-from-halo2-fork:
    if: ${{ github.event.label.name == 'benchmarks' }}

Reply:
Oh right! I missed the `if` line, thanks for the clarification! :)